storage/storage.go

```diff
@@ -0,0 +1,102 @@
+package storage
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"path"
+)
+
+const (
+	dirPerms  = 0777
+	filePerms = 777
```

Comment:
octal fileperms, don't make the files executable -> `0644` is usually fine.

Reply:
Done.